### PR TITLE
Add Makefile to build images

### PR DESCRIPTION
**Pull request by andypost**
This change makes easy to manually rebuild images

@todo We need to add ARGS

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+NAME = skilldlabs/frontend
+TAGS = d7 latest omega_4_3 omega_4_4 zen
```
Comment (andriyun) [inline Makefile]: LABEL directive was not added to d7 Docker file

File: latest/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,15 @@
 FROM mhart/alpine-node:7
 
-MAINTAINER Andy Postnikov
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-php" \
```
Comment (andriyun) [inline latest/Dockerfile]: wrong container name

File: omega_4_3/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,15 @@
 FROM ruby:2.1-alpine
-MAINTAINER Andy Postnikov
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-php" \
```
Comment (andriyun) [inline omega_4_3/Dockerfile]: wrong container name

File: omega_4_4/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,15 @@
 FROM ruby:2.1-alpine
-MAINTAINER Andy Postnikov
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-php" \
```
Comment (andriyun) [inline omega_4_4/Dockerfile]: wrong container name

File: zen/Dockerfile [hunk 5]
```diff
@@ -1,6 +1,15 @@
 FROM skilldlabs/frontend:latest
 
-MAINTAINER Andy Postnikov
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-php" \
```
Comment (andriyun) [inline zen/Dockerfile]: wrong container name

File: Makefile (hunk 1 above)
Comment (andypost) [inline Makefile]: ашчув

**Comment by andriyun** [comment]
It works!
Great

File: d7/Dockerfile [hunk 6]
```diff
@@ -1,5 +1,16 @@
 FROM mhart/alpine-node:7
 
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-gulp" \
```
Comment (andriyun) [inline d7/Dockerfile]: Why docker-gulp?
Regarding description on this page http://label-schema.org/rc1/#label-semantics
`Text description of the image. May contain up to 300 characters.`

File: latest/Dockerfile [hunk 7]
```diff
@@ -1,6 +1,15 @@
 FROM mhart/alpine-node:7
 
-MAINTAINER Andy Postnikov
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-gulp" \
```
Comment (andriyun) [inline latest/Dockerfile]: need proper descriprion

File: omega_4_3/Dockerfile [hunk 8]
```diff
@@ -1,5 +1,15 @@
 FROM ruby:2.1-alpine
-MAINTAINER Andy Postnikov
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-gulp" \
```
Comment (andriyun) [inline omega_4_3/Dockerfile]: need proper descriprion

File: omega_4_4/Dockerfile [hunk 9]
```diff
@@ -1,5 +1,15 @@
 FROM ruby:2.1-alpine
-MAINTAINER Andy Postnikov
+
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-gulp" \
```
Comment (andriyun) [inline omega_4_4/Dockerfile]: need proper descriprion

File: zen/Dockerfile [hunk 10]
```diff
@@ -1,6 +1,15 @@
 FROM skilldlabs/frontend:latest
 
-MAINTAINER Andy Postnikov
+ARG BUILD_DATE
+ARG VCS_REF
+
+LABEL org.label-schema.build-date=$BUILD_DATE \
+  org.label-schema.vcs-ref=$VCS_REF \
+  org.label-schema.schema-version="1.0" \
+  org.label-schema.name="docker-gulp" \
```
Comment (andriyun) [inline zen/Dockerfile]: need proper descriprion

**Comment by andriyun** [comment]
Ok for me
can  be merged

**Comment by andypost** [comment]
Does it needs reroll

**Comment by iberdinsky-skilld** [comment]
maybe add steps to check here? i'm not sure we need to update hub images for testing

File: Makefile [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+
+.PHONY: all build push
+
+all: build push
```
Comment (andriyun) [inline Makefile]: Related to flow described here #16 
Would be great don't push images before them could be tested.
* make (build images localy)
* some script or testing section 
* push image if checks passed

**Review by andriyun — CHANGES_REQUESTED** [review]
Need update Makefile related to flow what we are going to implement #16

File: Makefile (hunk 11 above)
Comment (andypost) [reply Makefile]: Any idea for testing? I think at least 2 goals in make file is enough so let's file separate PR for #16

Comment (iberdinsky-skilld) [reply Makefile]: https://github.com/skilld-labs/docker-frontend/issues/16#issuecomment-316282592

**Comment by andypost** [comment]
That should be in long time ago, just to make rebuild easy
The process of rebuild is manual now, this is just a helper
So I merging it & let's discus #16

**Pull request merged**